### ISSUE TITLE
fix: QG issue template format for easier use

### ISSUE THREAD

**Issue by almadigabor** [eclipse-tractusx/.github#14]
Currently it is really hard to use the [QG check issue template](https://github.com/eclipse-tractusx/.github/blob/main/.github/ISSUE_TEMPLATE/system-team-qg-checklist.md) for the following reasons:

- Every time and interaction happens (generating new issue from line, clicking the checkbox) all the details sections are collapsing
- Lines that have summary and details section can't be converted into new issues

1. For this purpose I would propose making this checklist simple making one line/check for each TRG that can be converted into an issue and in the new issue's description the problems can be detailed. We should also use this in parallel with the [tractusx-quality-checks](https://eclipse-tractusx.github.io/sig-release/) page that already covers some of the TRGs.
2. I would revisit the TRGs and make realistic assumption about what can be really checked and what is just a guide. For instance [TRG3.02](https://eclipse-tractusx.github.io/docs/release/trg-3/trg-3-2) only makes sense when know exactly the application's setup and which data should be persisted and which doesn't.

**Comment by FaGru3n** [comment]
already updated, only missing part is if it is ok for you with the back on topice section

https://github.com/eclipse-tractusx/.github/blob/main/.github/ISSUE_TEMPLATE/system-team-qg-checklist.md
